### PR TITLE
Update exchange intro text

### LINKED ISSUE #223: Update first run modal text with correct information
The current text in the first run modal on the exchange view has incorrect information, we should update this before launch.

### PR DESCRIPTION
**Pull request by lukechilds**
Resolves #223 

Updated the text intro to have correct information. Also linked to some external references.

Also added an `<ExternalLink>` component.

screen shot 2018-05-18 at 3 07 25 pm

### CODE REVIEW

File: app/renderer/components/ExternalLink.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import electron from 'electron';
+
+const open = electron.shell.openExternal;
```
Comment (sindresorhus) [inline app/renderer/components/ExternalLink.js]: This is shorter and more readable:

```js
const {openExternal} = electron.shell;
```

`open` is too generic.

Comment (lukechilds) [reply app/renderer/components/ExternalLink.js]: 👍

File: app/renderer/views/Exchange/Intro.js [hunk 2]
```diff
@@ -18,14 +19,15 @@ class Intro extends React.Component {
 		return (
 			<div>
 				<Modal
-					title="How order books work with HyperDEX"
-					open={this.state.shouldOpen}
+					title="How trading works with HyperDEX"
```
Comment (sindresorhus) [inline app/renderer/views/Exchange/Intro.js]: I think we should titlecapitalize words:

```
How Trading Works with HyperDEX
```

**Comment by sindresorhus** [comment]
`on chain` => `on-chain`

It looks weird only linking `UTXO` in `UTXOs`. You should link the whole word.

File: app/renderer/views/Exchange/Intro.js (hunk 2 above)
Comment (lukechilds) [reply app/renderer/views/Exchange/Intro.js]: Yeah, definitely, done 👍

**Comment by sindresorhus** [comment]
`trading a coin with` => `trading a currency with`?

**Comment by sindresorhus** [comment]
`This means it functions slightly differently to a centralised exchange.` => `This means it works slightly differently than a centralized exchange.`?

**Comment by lukechilds** [comment]
Thanks, good spots.

File: app/renderer/components/ExternalLink.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import electron from 'electron';
+import React from 'react';
+
+const {openExternal} = electron.shell;
+
+const ExternalLink = ({url, ...props}) => <a {...props} onClick={() => openExternal(url)}/>;
```
Comment (sindresorhus) [inline app/renderer/components/ExternalLink.js]: I think this should use the `Link` component instead, so it get's the `.Link` classname.

Comment (lukechilds) [reply app/renderer/components/ExternalLink.js]: I did originally but then it uses a `<button>` which messes up the styles when it's embedded in text. It won't allow any word breaks in the button element.

It makes sense for UI links to be buttons but I don't think it does for textual links in a sentence.

Comment (sindresorhus) [reply app/renderer/components/ExternalLink.js]: @lukechilds I've fixed this here: https://github.com/lukechilds/hyperdex/pull/234 It's now an `a` too.

**Comment by lukechilds** [comment]
Yummy

kapture 2018-05-18 at 16 30 21

**Pull request merged**